app/views/components/gds/gdsMainTemplate.scala.html

```diff
@@ -109,3 +109,21 @@
     footerItems = footer
   )(contentBlock)
 }
+
+<script type="text/javascript">
```

Comment:
Could/should this be in it's own file?
Should it be for Chrome browsers only?

Reply:
Maybe not browser specific.
I managed to replicate on QA using Safari

Reply:
@SteveSugden Good call.
I have also made use of ES6 arrow functions.

It also feels like it is only Firefox dealing with this issue and most browsers will have this problem.
Please review again.